src/entity/Profile.ts

```diff
@@ -6,14 +6,10 @@ export class Profile {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @Column()
-  name: string;
+  // should this be the primary column
+  @Column({ length: 40 })
```

Comment:
```suggestion
  @Column({ length: 40, unique: true })
```
if you choose to not use profileId as the pk for this table, at least make it unique

Reply:
making it pk